src/avfoundation.cs

```diff
@@ -741,6 +933,42 @@ interface AVAudioEngine {
 		[Notification]
 		[Field ("AVAudioEngineConfigurationChangeNotification")]
 		NSString ConfigurationChangeNotification { get; }
+
+		[TV (11, 0), NoWatch, Mac (10, 13), iOS (11, 0)]
+		[Export ("autoShutdownEnabled")]
+		bool IsAutoShutdownEnabled { [Bind ("isAutoShutdownEnabled")] get; set; }
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("enableManualRenderingMode:format:maximumFrameCount:error:")]
+		bool EnableManualRenderingMode (AVAudioEngineManualRenderingMode mode, AVAudioFormat pcmFormat, uint maximumFrameCount, out NSError outError);
```

Comment:
Swift name is the one used: https://developer.apple.com/documentation/avfoundation/avaudioengine/2881237-enablemanualrenderingmode